functions/__z.fish

```diff
@@ -35,6 +39,7 @@ function __z -d "Jump to a recent directory."
         printf "         -c --clean    Cleans out $Z_DATA\n"
```

Comment:
Maybe "Cleans out" is no longer a good description for `clean`.

Reply:
```
-c --clean  Remove paths that no longer valid in $Z_DATA
-p --purge Remove all paths in $Z_DATA
```
English is my second language, maybe you'll have a better description :)

Reply:
`Remove invalid paths in $Z_DATA` might be more succint, but that was an equally appropriate description (: